lib/decanter/parser/date_parser.rb

```diff
@@ -8,7 +8,11 @@ class DateParser < ValueParser
         raise Decanter::ParseError.new 'Expects a single value' if val.is_a? Array
         next if (val.nil? || val === '')
         parse_format = options.fetch(:parse_format, '%m/%d/%Y')
+        begin
         ::Date.strptime(val, parse_format)
+        rescue ArgumentError => e
+          raise Decanter::ValueFormatError.new 'invalid Date value for format' if e.message == "invalid date"
```

Comment:
If the message isn't "invalid date", should we pass it through? I believe right now it would be swallowed

Reply:
I believe for all `ArgumentErrors` it would be `"invalid date"` but I added a return raise incase it is not.